requirements.lock.txt

```diff
@@ -28,6 +28,7 @@ marshmallow-enum==1.5.1
 mypy-extensions==0.4.3
 natsort==8.1.0
 oauthlib==3.2.0
+packaging==21.3
```

Comment:
Can you also add this to `requirements-dev.lock.txt` too, just for consistency?

Reply:
https://github.com/Cray-HPE/sat/blob/feature/CRAYSAT-1385/requirements-dev.lock.txt#L37

Reply:
Oh, I guess so . Looks to me like it is actually a dependency of Sphinx (possibly others as well), which explains why it was in `requirements-dev.lock.txt` but not in `requirements.lock.txt`. Looks like they are the same version, which is reassuring as well.